android/app/build.gradle

```diff
@@ -74,18 +77,26 @@ android {
             debug.java.srcDirs += 'src/flipper/java'
         }
     }
+
+    testOptions {
+        unitTests.all {
+            if (!project.ext.isRntaProject) {
+                exclude "**/com/react/testapp/**"
+            }
+        }
+    }
```

Comment:
Would it be cleaner to exclude the tests in `.npmignore`?

Reply:
This doesn't exclude tests from `node_modules`, it is a flag to exclude a group of tests within the source directory. Tests are running directly against `rnta/android`, not against `example/node_modules/rnta/android`.

Reply:
Not quite sure I follow. If the tests are excluded from the published packages, do we still need this block here? Within RNTA, we'll still have access to them as intended.

Reply:
Oh, now I see! Yes, the approach with `.npmignore` will be better. I will take a look.